app/main/lib/shared_models/audio_model.py

```diff
@@ -38,7 +39,11 @@ def save(self, audio):
                 if audio.chromaprint_fingerprint is not None and not existing.chromaprint_fingerprint:
                     existing.chromaprint_fingerprint = audio.chromaprint_fingerprint
                     flag_modified(existing, 'chromaprint_fingerprint')
-                if audio.context not in existing.context:
+                if isinstance(existing.context, list) and audio.context not in existing.context:
```

Comment:
Let's check QA and Live. I believe `context` should always be a list of dictionaries. If it can be either, let's make the second block `elif` rather than `if`